homeassistant/components/overkiz/climate_entities/hitachi_air_to_air_heat_pump_ovp.py

```diff
@@ -106,6 +107,8 @@ def __init__(
             ClimateEntityFeature.TARGET_TEMPERATURE
             | ClimateEntityFeature.FAN_MODE
             | ClimateEntityFeature.PRESET_MODE
+            | ClimateEntityFeature.TURN_OFF
```

Comment:
As these are static they can be moved to class attributes instead.

Reply:
@gjohansson-ST are you sure? They are actually dynamic, because based on the device characteristics, the `ClimateEntityFeature.SWING_MODE` will be added to the list.

https://github.com/home-assistant/core/blob/664db681231b98de3bf37ffbc9b3913584920885/homeassistant/components/overkiz/climate_entities/hitachi_air_to_air_heat_pump_ovp.py#L115

Reply:
You're right. I missed that 👍